bundle/language_en.properties

```diff
@@ -632,6 +633,7 @@ angal.exatype.examtypebrowser.title
 angal.exatype.newexamtype.title                                                                        = New Exam Type
 angal.genericreportbill.doyouwanttoprintreceipt.msg                                                    = Do you want to print a receipt?
 angal.groupsbrowser.deletegroup.fmt.msg                                                                = Delete group: {0}?
+angal.groupsbrowser.softdeletegroup.fmt.msg                                                            = Could not delete the group '{0}' because he is used somewhere. Do you want to mark it as deleted ?
```

Comment:
```suggestion
angal.groupsbrowser.softdeletegroup.fmt.msg                                                            = Could not delete the group '{0}' because it is used somewhere. Do you want to mark it as deleted ?
```